### DISCUSSION TITLE
Como receber equipamentos da empresa?

### DISCUSSION THREAD

**Discussion by weltonrodrigo** [DyegoCosta/trabalhando-remoto#216] [General]
É preciso pagar imposto no computador?

**Comment by jpsantos1** [comment]
Até onde sei voce nao se preocupa com isso(impostos) e sim a empresa que vai fornecer o equipamento e podem mandar por correios, transportadora ou se morar proximoa um escritorio voce vai buscar.  Agora se a empresa for gringa até onde sei voce que tem que ter seu computador pra trabalhar pois seria inviavel ficar mandando e em alguns casos eles fornecem a grana toda ou ajuda de custo pra voce comprar um equipamento.
